react-native-libraries.json

```
@@ -13322,5 +13322,13 @@
     "android": true,
     "expo": true,
     "newArchitecture": true
-  }  
+  },
+  {
+    "githubUrl": "https://github.com/Kommunicate-io/Kommunicate-React-Native-SDK",
+    "examples": ["https://github.com/Kommunicate-io/Expo-Kommunicate-Sample-App"],
+    "npmPkg": "react-native-kommunicate-chat",
+    "iOS": true,
```

Thanks for recreating the PR with changes included! 👍 

```suggestion
    "ios": true,
```

Let's stick to the key name specified in schema:
* https://github.com/react-native-community/directory/blob/b64b73c45141234c3e6fdb188ab661a5cccace4a/react-native-libraries.schema.json#L25-L33

---

Updated